### PR TITLE
Blocks patients from viewing operating computer UI

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request
You can no longer view operating computer UI while lying down as to prevent patients from peeking at the computer.

## Why It's Good For The Game

Currently its very hard to pull off a surprise brainwashing surgery on someone as the patient can always peek at the computer to see what exactly is going on. With this change players will have to trust their doctors unless they want to be a victim of surgery related shenanigans.

## Changelog
:cl:
balance: You can no longer view operating computer UI while lying down
/:cl:

### CODE REVIEW

**Comment by SimplySmiley123** [comment]
Inb4 it changes nothing because no one trusts with surgery outside the main medbay treatment room.

**Comment by Sealed101** [comment]
it should have some kind of message to let the patient know that their grubby bedridden/otherwise prone hands aren't welcome on the computer

**Comment by Generubi** [comment]
i feel like this will only cause that patients get up in the middle of the surgery "just to check"

**Comment by SmArtKar** [comment]
> i feel like this will only cause that patients get up in the middle of the surgery "just to check"

I'm sure not all doctors would be happy having their time wasted like that, I have a couple of ideas if it becomes a big enough OOC issue though (doubt that)

File: code/modules/tgui/states/standing.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+GLOBAL_DATUM_INIT(standing_state, /datum/ui_state/not_incapacitated_state/standing, new)
+
+/datum/ui_state/not_incapacitated_state/standing
+
+/datum/ui_state/not_incapacitated_state/standing/can_use_topic(src_object, mob/user)
+	if (!isliving(user))
+		return ..()
+	var/mob/living/living_user = user
+	if (living_user.body_position)
```
Comment (MrMelbert) [inline code/modules/tgui/states/standing.dm:15]: Should probably check for `MOBILITY_STAND` instead so mobs without concepts of standing can be accounted for

**Comment by the-orange-cow** [comment]
> i feel like this will only cause that patients get up in the middle of the surgery "just to check"

New feature:  if you interrupt your own surgery you have a chance of botching it and causing permanent injuries.

**Comment by scrunklershrungle932** [comment]
doesn't medbay have anesthetic you can use for preventing them from peeking

**Pull request merged**